### PR TITLE
Updated teh message.

### PR DESCRIPTION
**Pull request by vshand11**
updated messages per request.

### CODE REVIEW

File: src/essential_validator.py [hunk 1]
```diff
@@ -392,38 +392,42 @@ def check_relationship(self, file_info, type, columns):
...
         def_rel_nodes = [ key for key in def_rel.keys()]
         rel_props_dic = {rel.split(".")[0]: rel.split(".")[1] for rel in columns if "." in rel}
         rel_props_dic_types = rel_props_dic.keys()
        
         #  check if missing relationship
         rel_missed = [node for node in def_rel_nodes if node not in rel_props_dic_types]
         if len(rel_props_dic_types) == 0:
-            msg = f'Relationships to parents, {json.dumps(rel_missed)}, are not specified.' if len(rel_missed) > 1 else f'Relationship to parent, "{rel_missed[0]}", is not specified.'
-            return False, f'“{file_info[FILE_NAME]}”: {msg}'
+            msg = f'Relationships to parents {json.dumps(rel_missed)} are not specified.' if len(rel_missed) > 1 else f'Relationship to parent "{rel_missed[0]}" is not specified.'
```
Comment (n2iw) [inline src/essential_validator.py:407]: I've never seen this message, will it still happen? What's the situation cause this error?

File: src/essential_validator.py [hunk 2]
```diff
@@ -392,38 +392,42 @@ def check_relationship(self, file_info, type, columns):
                 return True, None
             else: # check if invalid relationships
                 msg = f'Relationships {json.dumps(rel_props)} are not defined.' if len(rel_props) > 1 else f'Relationship "{rel_props[0]}" is not defined.'
```
Comment (n2iw) [inline src/essential_validator.py:394]: I've never seen this message, will it still happen? What's the situation cause this error?

Comment (vshand11) [reply src/essential_validator.py:394]: A tsv file with invalid relationship(s) while no parents defined in data model.   e.g.  add relationship(s) in a tsv file for program node type, you eill see the message.

File: src/essential_validator.py (hunk 1 above)
Comment (vshand11) [reply src/essential_validator.py:407]: updated

File: src/essential_validator.py (hunk 2 above)
Comment (n2iw) [reply src/essential_validator.py:394]: In that case, let's change the message to "**Relationship column "test.test_id" is invalid, "program" node should not have any relationship columns.**". Also, please split it into multiple errors if there are multiple relationship columns.

**Review by n2iw — CHANGES_REQUESTED** [review]

File: src/essential_validator.py (hunk 2 above)
Comment (vshand11) [reply src/essential_validator.py]: updated

**Review by n2iw — APPROVED** [review]
LGTM

**Pull request merged**